<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GirisEkrani</class>
 <widget class="QMainWindow" name="GirisEkrani">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>2000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Giriş Ekranı</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>930</y>
      <width>461</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>25</pointsize>
         <weight>9</weight>
         <italic>false</italic>
         <bold>false</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 25pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Kullanıcı Adı:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>25</pointsize>
         <weight>9</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 25pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Şifre:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>930</y>
      <width>391</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLineEdit" name="kullanici">
       <property name="styleSheet">
        <string notr="true">font: 75 15pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="sifre">
       <property name="styleSheet">
        <string notr="true">font: 75 15pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>1410</y>
      <width>751</width>
      <height>451</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="font">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>20</pointsize>
         <weight>9</weight>
         <italic>false</italic>
         <bold>false</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(90, 101, 127);
font: 75 20pt &quot;MS Shell Dlg 2&quot;;
</string>
       </property>
       <property name="text">
        <string>Giriş Yap</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="font">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>20</pointsize>
         <weight>9</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(15, 15, 15);
font: 75 20pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(90, 101, 127);</string>
       </property>
       <property name="text">
        <string>Kaydol</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>260</y>
      <width>751</width>
      <height>321</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ÜretSen</string>
    </property>
    <property name="icon">
     <iconset>
      <normalon>:/icon/Yeni klasör/üretSen.jpeg</normalon>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>300</width>
      <height>300</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
